src/libraries/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -121,6 +122,7 @@ public void DynamicDirectory_Null()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/38994", TestPlatforms.Browser)]
```

Comment:
You can just fix this one by replacing `Assembly.GetExecutingAssembly().GetName().Name;` below with `DefaultDomain`.

Reply:
Thanks!